tests/test_aggregates.py

```diff
@@ -250,14 +250,14 @@ def test_candidate_aggregates_by_committee(self):
             results = self._results(
                 api.url_for(
                     resource,
-                    committee_id=self.committee.committee_id,
```

Comment:
The tests are passing for me with committee_id. Should we keep it committee_id since the test is by_committee?

Reply:
That makes sense! I'll put in another ticket for this work. Thanks @tmpayton!